frontend/src/js/KonvaManager/EdgeManager.ts

```diff
@@ -2,6 +2,16 @@ import Konva from "konva";
 import DraggableManager from "./DraggableManager";
 import { Cordinates, HighlightConfig, Vertex } from "./VertexManager";
 
+export interface EdgeDTO extends Konva.LineConfig {
```

Comment:
What does DTO stand for?

Reply:
Its data transfer object, actually we should use them always when we send something bc it creates abstraction layer that makes communication more modular. Since this interface extends konva class its not truely  dto but i will change that soon, I hope :D